script.js

```diff
@@ -0,0 +1,621 @@
+  nameEl.innerText = cardData.display;
+
+  const cardEl = document.createElement('div');
+  cardEl.classList.add('card', 'col-3', 'align-items-center', 'col-md-2');
+
+  cardEl.appendChild(nameEl);
+  cardEl.appendChild(suitEl);
+
+  return cardEl;
+};
+
+// For exchanging cards -------------------------------------------
+// select the card to exchange or unselect it
+const selectOrUnselectCardToExchange = (cardEl, cardToExchange) => {
+  // when player clicks this card and it has not been selected before,
+  // store it in an array of cards that will be exchanged
+  // but if card is selected before,
+  // remove it from the array of cards that will be exchanged.
+  let isCardPresent = false; // false if card has not been selected before
+  if (cardsToExchange.length > 0) { // only check if there are cards in array
+    for (let j = 0; j < cardsToExchange.length; j += 1) {
+      if (cardToExchange === cardsToExchange[j]) {
+        isCardPresent = true;
+        cardsToExchange.splice(j, 1); // remove the card from array
+        j -= 1; // account for the decrease in array length
+
+        // remove the card border display to let player know card is dis-selected
+        cardEl.classList.remove('card-border');
+      }
+    }
+  }
+  if (isCardPresent === false) {
+    cardsToExchange.push(cardToExchange); // store the card
+
+    // display the card border to let player know card is selected
+    cardEl.classList.add('card-border');
+  }
+};
+// exchange cards
+const exchangeCards = () => {
+  // exchange the selected cards in playerHand
+  for (let i = 0; i < playerHand.length; i += 1) {
+    for (let j = 0; j < cardsToExchange.length; j += 1) {
+      if (cardsToExchange[j].rank === playerHand[i].rank
+        && cardsToExchange[j].suit === playerHand[i].suit) {
+        playerHand.splice(i, 1, deck.pop());
+      }
+    }
+  }
+  // empty cardsToExchange array since we do not need the cards inside anymore
+  cardsToExchange = [];
+
+  // clear previous display of player's hand
+  playerHandContainer.innerHTML = '';
+  // make the player's cards' display and display them
+  for (let i = 0; i < playerHand.length; i += 1) {
+    const cardEl = makeCardElement(playerHand[i]);
+    playerHandContainer.appendChild(cardEl);
+  }
+};
+
+// For calculating player hand score -------------------------------
+// reorder player's cards from highest to lowest rank
+const reorderCards = () => {
+  /** for each position starting from the 0th index
+   * check cards in positions further down the array for higher ranks
+   * and swap the cards in those positions */
+  let j = 1;
+  for (let i = 0; i < (playerHand.length - 1); i += 1) {
+    for (let k = j; k < playerHand.length; k += 1) {
+      if (playerHand[k].rank > playerHand[i].rank) {
+        const lowerRankCard = playerHand[i];
+        const higherRankCard = playerHand[k];
+        playerHand[i] = higherRankCard;
+        playerHand[k] = lowerRankCard;
+      }
+    }
+    j += 1;
+  }
+};
```

Comment:
Newlines between functions for readability. If you want to separate groups of functions, consider adding a multi-line comment and/or putting 2 newlines between groups.